tests/phpunit/tests/html-api/wpHtmlProcessor-bookmark.php

```diff
@@ -157,4 +157,15 @@ public static function data_processor_constructors(): array {
 			'Fragment parser' => array( array( WP_HTML_Processor::class, 'create_fragment' ) ),
 		);
 	}
+
+	/**
+	 * @ticket 62521
+	 */
```

Comment:
This will require some updated when `_doing_it_wrong` call gets included, see a similar test:

https://github.com/WordPress/wordpress-develop/blob/0d428194d1e6d6f1285d45cafd84b18f514070c1/tests/phpunit/tests/html-api/wpHtmlTagProcessor-bookmark.php#L416-L417